### PR TITLE
Remove Person 

### LINKED ISSUE #148: Remove Person from AB3

### PR DESCRIPTION
**Pull request by kormingsoon**

### CODE REVIEW

File: src/main/java/seedu/address/AppParameters.java [hunk 1]
```diff
@@ -45,20 +45,7 @@ public static AppParameters parse(Application.Parameters parameters) {
...
-        }
-
         AppParameters otherAppParameters = (AppParameters) other;
         return Objects.equals(getConfigPath(), otherAppParameters.getConfigPath());
     }
-
-    @Override
-    public int hashCode() {
-        return configPath.hashCode();
-    }
```
Comment (Marcon2509) [inline src/main/java/seedu/address/AppParameters.java:63]: wait why is this needed to be removed. isnt this just to make sure an app parameter object is the same as another app parameter with the same stuff?

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -103,45 +98,37 @@ public void init() throws Exception {
...
-                logger.info("Data file not found. Will be starting with a sample bidderAddressBook");
+                logger.info("Bidder Data file not found. Will be starting with a sample bidderAddressBook");
             }
             if (!sellerAddressBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample sellerAddressBook");
+                logger.info("Seller Data file not found. Will be starting with a sample sellerAddressBook");
             }
             if (!meetingBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample meetingBook");
+                logger.info("Meeting Data file not found. Will be starting with a sample meetingBook");
```
Comment (Marcon2509) [inline src/main/java/seedu/address/MainApp.java:130]: I think it shud be BidderAddressBook file not found. or MeetingBook cause it the bidderbook thats not found not the data on the bidders

File: src/test/java/seedu/address/model/ModelManagerTest.java [hunk 3]
```diff
@@ -137,12 +101,6 @@ public void hasBid_bidNotInBidBook_returnsFalse() {
         assertFalse(modelManager.hasBid(BID_A));
     }
 
-    @Test
-    public void hasBid_bidInBidBook_returnsTrue() {
-        modelManager.addPerson(ALICE);
-        assertTrue(modelManager.hasPerson(ALICE));
-    }
-
```
Comment (Marcon2509) [inline src/test/java/seedu/address/model/ModelManagerTest.java:145]: huh weird test not sure what i doing for that lol

**Review by Marcon2509 — APPROVED** [review]
LGTM for Bid side

**Review by dianneloh9 — APPROVED** [review]

**Review by Christopher-LM — APPROVED** [review]
